packages/common/src/api/tan-query/useFeaturedProfiles.ts

```diff
@@ -0,0 +1,16 @@
+import { useExploreContent } from './useExploreContent'
+import { useUsers } from './useUsers'
+
+type Config = {
+  staleTime?: number
+  enabled?: boolean
+}
+
+export const useFeaturedProfiles = (config?: Config) => {
+  const { data: exploreContent } = useExploreContent(config)
+  return useUsers(exploreContent?.featuredProfiles, {
```

Comment:
any reason we don't need the limit slice here? are there just fewer of them?

Reply:
There was a limit call in the UI so this is a bit more performant to only fetch what we need